node/src/BaseClient.ts

```diff
@@ -381,17 +382,17 @@ export class BaseClient {
     ) {
         const message = Array.isArray(command)
             ? redis_request.RedisRequest.create({
-                  callbackIdx,
```

Comment:
lint error?

Reply:
Do you know how to reconfigure VS code to stop code linting?

Reply:
Fixed